sample/index.js

```diff
@@ -176,5 +214,49 @@ const bulkDeleteFiles = async (imagekitInstance, fileIds) => {
     return "success";
 }
 
+const bulkAddTags = async (imagekitInstance, fileIds, tags) => {
+    const response = await imagekitInstance.bulkAddTags(fileIds, tags);
+    return "success";
+}
+
+const bulkRemoveTags = async (imagekitInstance, fileIds, tags) => {
+    const response = await imagekitInstance.bulkRemoveTags(fileIds, tags);
+    return "success";
+}
+
+const copyFile = async (imagekitInstance, sourceFilePath, destinationPath) => {
+    const response = await imagekitInstance.copyFile(sourceFilePath, destinationPath);
+    return "success";
+}
+
+const moveFile = async (imagekitInstance, sourceFilePath, destinationPath) => {
+    const response = await imagekitInstance.moveFile(sourceFilePath, destinationPath);
```

Comment:
Getting error in this function. Since the sample uploads few images, the path could vary, but it should work. Create a test account and run `index/sample.js` file to test it.
```
Bulk add tags response:  success 

Bulk remove tags response:  success 

Folder creation response:  success 

File copy response:  success 

Encounterted Error:  {
  "message": "No file found with filePath /folder1//sample-folder-sdk/ruby/ruby-sdk-testing_VdTeODis5.png",
  "help": "For support kindly contact us at support@imagekit.io .",
  "reason": "SOURCE_FILE_MISSING"
}
```